coordinator/src/main/scala/filodb.coordinator/ShardMapper.scala

```diff
@@ -276,4 +293,14 @@ private[filodb] object ShardMapper extends StrictLogging {
 
   final case class ShardError(event: ShardEvent, context: String)
     extends Exception(s"$context [shard=${event.shard}, event=$event]")
+
+  def statusToLetter(status: ShardStatus): String = status match {
+    case ShardStatusUnassigned  => "."
+    case ShardStatusAssigned    => "N"
+    case ShardStatusActive      => "A"
+    case ShardStatusError       => "E"
+    case s: ShardStatusRecovery => "R"
```

Comment:
Eh? Why the 's'?

Reply:
`ShardStatusRecovery` is a `case class` with % recovered.

Reply:
The compiler complains otherwise?

Reply:
Yeah `case SomeClass =>` can only be used for objects/singletons